packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -246,6 +255,10 @@ public void init(
         if (!Strings.isNullOrEmpty(hostedDomain)) {
           optionsBuilder.setHostedDomain(hostedDomain);
         }
+        if (!Strings.isNullOrEmpty(serverClientId)) {
+          optionsBuilder.requestServerAuthCode(serverClientId, true);
+          optionsBuilder.requestIdToken(serverClientId);
```

Comment:
Is this required? Why do we not have the same on iOS? What happens if this block also calls `requestIdToken`?
```dart
        if (clientIdIdentifier != 0) {
          optionsBuilder.requestIdToken(registrar.context().getString(clientIdIdentifier));
        }
```